charts/opentelemetry-operator/templates/clusterrole.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.rbac.create }}
```

Comment:
There are multiple ClusterRoles defined in this file.  Is there value in being able to enabled/disable each cluster role individually?  Or does it not ever make sense with the operator to have 1 cluster role but not the others.

Reply:
I would have the flag to switch between all or nothing. The issue in a multi tenant k8s cluster is that you cannot deploy ClusterRole at all.